### ISSUE TITLE
LaTeX to RASH conversion

### ISSUE THREAD

**Issue by essepuntato** [essepuntato/rash#58]
Creating a script for automatising the conversion from LaTeX (considering at least the Springer LNCS and the ACM ICPS styles) into RASH.

**Comment by gcpdev** [comment]
@essepuntato I need to state that this is really needed. Authors usually have their sharelatex paper, and converting it to RASH is a very time-demanding and stressful activity: converting the PDF to .docx/.odt, struggling with Word / LibreOffice formats to get rid of non-conforming XML's and so on. It is bad to authors, especially if you consider that they usually work in the final version of their papers close to the deadline.

**Comment by essepuntato** [comment]
@gcpdev 

I totally agree. The point is that the task is quite difficult to automatize, and, in addition, I haven't found any person/student who is interested in helping with this yet.

**Comment by artreven** [comment]
Hi,

Just to confirm that this is still needed. I wanted to use your tool for preparing a submission. I prepare my papers in LaTeX and I already have some basis (from a previous submission). Unfortunately, I am not going to use your tool this time, but I will surely consider using it next time if this issue will be resolved.
Thank you for your great work!
